### DISCUSSION TITLE
model-viewer showing blank page or reloading on IOS 15 (and 14), white model on some Android phone

### DISCUSSION THREAD

**Discussion by royalboi** [google/model-viewer#2921] [Q&A] [answered]
Hi everyone, I'm doing 3D scanner and i'm not a developer and i have a limited knoweger about coding but i still can understand some of them.

Recently i start using model-viewer by upload it to https://modelviewer.dev/editor/ then upload to my own server and its work perfectly on my PC 

But when i send link to my client, they said they cannot see the texture, just the model with white texture (he's using Xiaomi phone with Android 10)
![image](https://user-images.githubusercontent.com/61173679/138876778-ad0ffd6f-9f23-44a1-87b3-89edadca41d8.png)

Some report that the page keep refresh over and over (Iphone XR-ios 14 and Iphone 12PM-ios 15)

Iphone12PM
![image](https://user-images.githubusercontent.com/61173679/138876721-958401f6-abf6-421c-bb64-54d271ae6768.png)

I test on my phone and i dont know why but it's work perfectly fine (Samsung S10P-Android 11, Iphone7P - IOS13)
Samsung S10
![image](https://user-images.githubusercontent.com/61173679/138876355-3edc9aa0-09cf-4e31-bf07-c9b45dd328fd.png)

Iphone 7P
![image](https://user-images.githubusercontent.com/61173679/138876638-9d814e34-3543-4367-ac6b-07c5bbc8627c.png)

I deciced to post here first because i don't know it's a bug or my mistake
Here the model that i made

https://spiky-ionized-phlox.glitch.me/

Thanks for reading.

**Comment by elalish** [comment, marked as answer]
Hmm, it's crashing webGL on my macbook too. It  looks like your textures are  4096x4096. That's very large (and why your GLB is so big) and I'd guess we're just running out of GPU memory for this process (which will be OS/device specific). I'd recommend downsampling your textures (you can  probably make them a lot  smaller without losing noticeable  quality). If you can't bare that, try encoding them as KTX2 textures, since that will save considerable GPU memory (and make them decoder faster).

**Reply by royalboi** [reply]
Thank you, i will try that
